db/migrate/20200823172649_add_head_reference_to_pull_requests.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+class AddHeadReferenceToPullRequests < ActiveRecord::Migration[6.0]
+  def change
+    add_reference :pull_requests, :head
```

Comment:
There's a bunch of tables that are migrated several times, for instance this `add_reference` could totally be part of the previous `RecreateShipitPullRequests`. I understand that this is a dump of an iterative development, but if we're to merge this one shot it would be worth merging merging the migrations on a per table basis (one for each table).